tests/data/exception_skill/behaviours.py

```diff
@@ -18,3 +18,22 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the behaviours for the 'exception' skill."""
+from aea.skills.base import Behaviour
+
+
+class ExceptionBehaviour(Behaviour):
```

Comment:
replacement of `ExceptionTask`. It couldn't be used anymore - the `execute()` method is not called automatically by the main loop.